.githooks/pre-commit

```diff
@@ -57,4 +57,6 @@ done
 # If there are whitespace errors, print the offending file names and fail.
 git diff-index --check --cached $against --
 
+npm shrinkwrap --dev
```

Comment:
Could you elaborate? 

I usually prefer to know which version of my dependencies I'm going to run.